docs/versioned-plugins/inputs/file-v4.4.0.asciidoc

```diff
@@ -533,4 +538,9 @@ Supported values: `us` `usec` `usecs`, e.g. "600 us", "800 usec", "900 usecs"
 [NOTE]
 `micro` `micros` and `microseconds` are not supported
 
+ifeval::["{versioned_docs}"=="true"]
+:branch: current
+:ecs_version: current
+endif::[]
```

Comment:
Clearing variables at the end of each file will be required until I finish cleaning up all existing files.  After cleanup, this step won't be required and I can remove this from the each plugin source file.